Formula/f/fwupd.rb

```diff
@@ -0,0 +1,67 @@
+class Fwupd < Formula
+  include Language::Python::Shebang
+  include Language::Python::Virtualenv
+
+  desc "Firmware update daemon"
+  homepage "https://github.com/fwupd/fwupd"
+  url "https://github.com/fwupd/fwupd/releases/download/1.9.5/fwupd-1.9.5.tar.xz"
+  sha256 "201ec20538044749c96dbff708e52f7d79038de787b90a2762e76793080aa37c"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/fwupd/fwupd.git", branch: "main"
+
+  depends_on "gcab" => :build
+  depends_on "gi-docgen" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.11" => :build
```

Comment:
Just a hunch something like this could make the CI pass, as it's installing Jinja2 with `--no-deps`, while Jinja2 requires "MarkupSafe>=2.0"
```suggestion
  depends_on "python@3.11" => :build
  depends_on "python-markupsafe" => :build
```

Reply:
@martinhoyer this is entirely plausible!

> It seems I've installed jinja2 on my test machine using pip

...which probably installed MarkupSafe at the same time!I've rebased against master and tried your suggestion -- fingers crossed!

Reply:
My bad
`FormulaAudit/DependencyOrder: dependency "python-markupsafe" (line 19) should be put before dependency "python@3.11" (line 27)`

Reply:
That audit error is a bug in the last commit for brew, it'll go away soon

Reply:
> apparently that's provided by [gettext](https://formulae.brew.sh/formula/gettext)

Added as a `depends_on` as I think we need this for runtime too, thanks. I've also moved gcab from a `:build`-only to a full `depends_on` as we need it at runtime too.

Reply:
@SMillerDev do i need to rebase against master and re-push or do I just wait and re-run the CI? Thanks.

Reply:
There is also this on Linux:
`plugins/uefi-capsule/meson.build:103:15: ERROR: Dependency 'fwupd-efi' is required but not found.`

Reply:
Yup, I'll disable that the next time I push; I don't want to spin up the CI for no reason if I have to wait until tmw for new containers for instance.

Reply:
I've manually disabled the two plugins with compile failures on Linux -- fixed properly upstream in https://github.com/fwupd/fwupd/pull/6147